config/development_oauth.yaml

```diff
@@ -0,0 +1,8 @@
+authorization:
```

Comment:
Like others, you need to copy the whole file from https://github.com/uber/cadence/blob/master/config/development.yaml and then add this authorization section. 

The `config/development_oauth.yaml` will only override `base.yaml` which only have a section of logging config.

Reply:
actually I have to remove some section in 
```
archival:
  history:
    status: "enabled"
    enableRead: true
    provider:
      filestore:
        fileMode: "0666"
        dirMode: "0766"
      gstorage:
        credentialsPath: "/tmp/gcloud/keyfile.json"
  visibility:
    status: "enabled"
    enableRead: true
    provider:
      filestore:
        fileMode: "0666"
        dirMode: "0766"

domainDefaults:
  archival:
    history:
      status: "enabled"
      URI: "file:///tmp/cadence_archival/development"
    visibility:
      status: "enabled"
      URI: "file:///tmp/cadence_vis_archival/development"

```
And also add 
```
system.enableParentClosePolicyWorker:
  - value: false
    constraints: {}
system.enableWorkflowShadower:
  - value: false
    constraints: {}
system.enableFailoverManager:
  - value: false
    constraints: {}
```

So that we don't start any system workflow for now. Because we don't have a way to authorize those request unitil golang client authorizer is implemented. 
I have address those comments myself and pushed to your branch.